### DISCUSSION TITLE
[Polygon x Truffle] tx fee (1.43 ether) exceeds the configured cap (1.00 ether)

### DISCUSSION THREAD

**Discussion by vfabing** [ConsenSys-archive/truffle#5916] [General Q&A]
Hi,

I noticed that recently the gas price on Polygon has increased a lot (moving from 30-40 gwei at the lowest to 150 gwei at the moment, which is 4-5 times higher than before!)

I got the following error message when using `truffle migrate` on Polygon network:
`tx fee (1.43 ether) exceeds the configured cap (1.00 ether)`

I was wondering if there is any way to modify the "configured cap"?
Because if not, it feels like that some smart contract that were deployed few months ago would now not be able to be updated/deployed anymore, and would need to be further split / optimized... 🤔

In any case, thank you very much for your amazing work, and looking forward to your kind guidance on this matter 🙇

**Comment by LoveKosmos** [comment]
Hi there, 

What is your gas and gasPrice set to in the truffle-config?
Would you have a test repo to share with us?

**Comment by vfabing** [comment]
Hi @LoveKosmos!

gasPrice was `220e9` without gas set (tried to set the gas to `6e6` but didn't change anything).

Also I could not reproduce the error on testnet but only on mainnet (where gas price is higher I guess)

Sorry I don't have a repro repository at the moment 🙇 I guess any "big enough" smart contract would do the job 🤔

**Reply by justindorber** [reply]
Any news on this @vfabing?
